src/adapters/auth.ts

```diff
@@ -20,6 +21,23 @@ class AuthAdapter {
 
     return requestManager('POST', `${process.env.REACT_APP_NIMBLE_SURVEY_BASE_URL}/api/v1/oauth/token`, requestOptions)
   }
+
+  static forgotPassword = (email: string): Promise<AxiosResponse> => {
+    const requestOptions = {
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      data: {
+        user: {
+          email: email
+        },
+        client_id: process.env.REACT_APP_CLIENT_ID,
```

Comment:
Let's create a function for this default values -

```
  static get DEFAULT_PAYLOAD() {
    return {
      client_id: process.env.REACT_APP_CLIENT_ID,
      client_secret: process.env.REACT_APP_CLIENT_SECRET
    };
  }
```

and use it here like `...this.DEFAULT_PAYLOAD`

Reply:
Good idea. Fixed in 117eaa3ff205ed78d945b4ea1e86eba7f54a05b7
We cannot use `this` with `static`

> 'this' cannot be referenced in a static property initializer.ts(2334)